beacon-chain/sync/validate_beacon_blocks_test.go

```diff
@@ -98,7 +98,6 @@ func TestValidateBeaconBlockPubSub_BlockAlreadyPresentInCache(t *testing.T) {
 		t.Fatal(err)
 	}
 	msg := &ethpb.BeaconBlock{
-		Slot:       1,
```

Comment:
Why these changes in `beacon-chain/sync/validate_beacon_blocks_test.go`?

Reply:
With slot 1, we get an error ` could not process slot from the future, slot time 1572483483 > current time 1572483471`.
Which makes sense because `genesis_time` is `time.Now` and we are passing in a block with slot 1 right away. This doesn't happen with 6s slot but only with 12s slot